facturacion_mexico/fiscal_state/ffm_state.py

```
@@ -0,0 +1,251 @@
+					filters={"name": ["in", pe_names], "docstatus": 1},
+					fields=["name"],
+					limit=1,
+				)
+				has_active_pe = bool(submitted_pe)
+
+	facts["has_active_payment_entry"] = has_active_pe
+
+	# ── Tax system (para validar si puede timbrar) ────────────────────────
+	tax_system = ffm.get("fm_tax_system") or ""
+	facts["tax_system_valid"] = bool(tax_system) and not (
+		tax_system.startswith("⚠️") or tax_system.startswith("❌")
```

Ruff (0.15.12)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)